content/en/docs/setup/install/virtual-machine/index.md

```diff
@@ -438,6 +438,14 @@ Run the following commands on the virtual machine you want to add to the Istio m
     Hello version: v1, instance: helloworld-v1-578dd69f69-fxwwk
     {{< /text >}}
 
+## Next Steps
+
+For more information about virtual machines:
+
+* [Virtual Machine Architecture](/docs/ops/deployment/vm-architecture/) to learn about the high level architecture of Istio's virtual machine integration.
```

Comment:
This one doesn't seem like a Next Step, seems like something they should be able to read about at the top? I'd move this one up into the intro to this page.